### DISCUSSION TITLE
Disable Copilot chat modal on opened untitled file

### DISCUSSION THREAD

**Discussion by nhuxhr** [community/community#67236] [Copilot Conversations]
The copilot stays even after editing the untitled file. I don't like the behavior. How do I disable it?

![IMG_20230915_213556_507.png](https://github.com/community/community/assets/51025764/ce158c8c-b8e2-404c-97a2-622870b7dcad)

**Comment by gmeligio** [comment]
I have the same request. Here is the link to an older discussion related to this. I referenced this discussion there too. 

https://github.com/orgs/community/discussions/11254

**Comment by miskr-instructure** [comment]
+1

Disabling copilot by language doesn't cover this, because the language of Untitled "files" is guessed by VS Code (and can change a you type). 

Disabling for plain text might seem like a good idea, but the Untitled "file" will not stay "Plain Text" as you start typing (eg. type SELECT and it infers SQL). The only way to ensure it stays Plain Text (for which one might have disabled copilot) is manually selecting Plain Text language at the bottom right corner, then VS Code will stop trying to guess.
